packages/aws-cdk/test/integ/helpers/cdk.ts

```diff
@@ -722,9 +723,7 @@ const installNpm7 = memoize0(async (): Promise<string> => {
   await shell(['rm', '-rf', installDir]);
   await shell(['mkdir', '-p', installDir]);
 
-  await shell(['npm', 'install',
-    '--prefix', installDir,
-    'npm@7']);
+  await shell(['npm', 'install', 'npm@7'], { cwd: installDir });
```

Comment:
While we're there...

```suggestion
  await shell(['npm', 'install', 'npm@8'], { cwd: installDir });
```

Reply:
That's another can of worms I'd rather not get into in this PR.